README.md

```diff
@@ -340,6 +340,43 @@ store.dispatch('dec')        // Unknown event
 In order to work properly for imports, it is considering adding
 `allowSyntheticDefaultImports: true` to `tsconfig.json`.
 
+## Server-side rendering
+
+In order to preload data for server-side rendering, Storeon provide `customContext` function to create your own `useStoreon` hooks that it will depends on your custom context.
```

Comment:
We need to keep 80 column limit here as well

Reply:
Patched 7709608cba35ac0612d37b9ef43d5dc2f2368753